### PR TITLE
Units dead

### PR DESCRIPTION
**Pull request by PatrikLundell**
See discussion in df-structures issue 247.
Link:  DFHack/df-structures#247

### CODE REVIEW

File: library/modules/Units.cpp [hunk 1]
```diff
@@ -1568,6 +1568,27 @@ bool Units::isUndead(df::unit* unit)
              && !unit->curse.add_tags1.bits.BLOODSUCKER ));
 }
 
+bool Units::isGhost(df::unit *unit)
+{
+    CHECK_NULL_POINTER(unit);
+
+    return !unit->flags3.bits.ghostly;
```
Comment (lethosor) [inline library/modules/Units.cpp]: Is this inverted?

**Comment by PatrikLundell** [comment]
Blast! Yes, it was.

**Comment by lethosor** [comment]
Merged in 10e6016 (I squashed your 4 commits to individual files into one commit with the same message, but that changed the hash of all of the commits, so GitHub didn't auto-close this PR.)

**Comment by PatrikLundell** [comment]
That's the reason I left an issue link in human but not machine readable format originally for these PRs: I thought it might result in a premature auto close.

**Comment by lethosor** [comment]
It only closes it automatically if the link is immediately preceded by "fix", "close", "resolve", or other forms of those words. There's a GitHub help page about it.

**Pull request closed without merging**